sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteFilesTest.java

```diff
@@ -477,6 +480,39 @@ public void testBuildWrite() {
     assertThat(write.getComputeNumShards(), equalTo(originalSharding));
   }
 
+  @Test
+  public void testWriteTransformName() {
```

Comment:
Anywhere in runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow we can put these tests? It is very specific for dataflow that I don't feel like it is good idea to put it under sdk.

Also can we create a github issue as a TODO for the tests for a more generic solution for identifying these transforms when submitting to dataflow via v1b3 API. We should remove these tests after we have a more safe solution.